src/main/java/io/lighty/yang/validator/formats/JsTree.java

```diff
@@ -55,145 +55,182 @@
-                            qnamesCopy.remove(integerListIterator.previous().intValue());
-                        }
-                        if (!this.schemaContext.findDataTreeChild(qnamesCopy).isPresent()) {
-                            removeChoiceQnames.add(i - 1);
-                        } else if (this.schemaContext.findDataTreeChild(qnamesCopy).get()
-                                instanceof ActionNodeContainer) {
-                            final ActionNodeContainer actionSchemaNode =
-                                    (ActionNodeContainer) this.schemaContext.findDataTreeChild(qnamesCopy).get();
-                            actions = actionSchemaNode.getActions();
-                        }
-                    }
-                    ids.add(modelAugmentationNumber++);
-                    HtmlLine line = new HtmlLine(new ArrayList<>(ids), node, inputOutputOther, this.schemaContext,
-                            removeChoiceQnames, namespacePrefix, Optional.empty(), false);
-                    lines.add(line);
-                    resolveChildNodes(lines, new ArrayList<>(ids), node, RpcInputOutput.OTHER, removeChoiceQnames,
-                            Collections.emptyList());
-                    ids.remove(ids.size() - 1);
-                }
-                for (Line line : lines) {
-                    final String linesText = line.toString();
-                    LOG.info(linesText);
-                }
-                lines = new ArrayList<>();
+                printLines(getAugmentationNodesLines(singletonListInitializer.getSingletonListWithIncreasedValue(),
+                                                     augNode, removeChoiceQnames));
             }
-            // rpcs
-            for (RpcDefinition node : module.getRpcs()) {
-                final ArrayList<Integer> rpcId = new ArrayList<>(Collections.singletonList(id++));
-                HtmlLine htmlLine = new HtmlLine(rpcId, node, RpcInputOutput.OTHER, this.schemaContext,
-                        removeChoiceQnames, namespacePrefix, Optional.empty(), false);
+
+            // Rpcs
+            printLines(getRpcsLines(singletonListInitializer, module, removeChoiceQnames));
+
+            // Notifications
+            printLines(getNotificationsLines(singletonListInitializer, module, removeChoiceQnames));
+        }
+
+        LOG.info("</table>");
+        LOG.info("</div>");
+        String loadJS = loadJS();
+        LOG.info("{}", loadJS);
+        LOG.info("</body>");
+        LOG.info("</html>");
+    }
+
+    private void printLines(final List<Line> lines) {
+        for (Line line : lines) {
+            final String linesText = line.toString();
+            LOG.info("{}", linesText);
+        }
+    }
+
+    private List<Line> getNotificationsLines(final SingletonListInitializer singletonListInitializer,
+                                             final Module module, final List<Integer> removeChoiceQnames) {
```

Comment:
same,
please use continuation indent: `8`, do not align to first parameter

Reply:
done